modules/ROOT/pages/shared-bucket-access.adoc

```diff
@@ -55,7 +55,7 @@ The import process is a key part of mobile convergence.
 It is the means by which Sync Gateway becomes aware of non-Sync Gateway data changes and obtains the mobile metadata it requires to replicate changes.
 Any link:glossary.adoc#non-sync-gateway[non-Sync Gateway change] is eligible for import.
 
-NOTE: By default Sync Gateway distributes import processing across all nodes in the Sync Gateway cluster
+NOTE: By default import processing is switched-off and should only be enabled on one node
```

Comment:
Should we add something version specific, just in case folks don't doublecheck? Or is that messing up the general approach?

```suggestion
NOTE: By default import processing is disabled in version 2.6 and earlier and should only be enabled on one node
```